Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -848,35 +854,28 @@ defineSuite([
             expect(stats.numberOfPendingRequests).toEqual(1);
             scene.primitives.remove(tileset);
 
-            return root.contentReadyPromise.then(function(root) {
-                fail('should not resolve');
-            }).otherwise(function(error) {
-                // Expect the root to not have added any children from the external tileset.json
-                expect(root.children.length).toEqual(0);
-                expect(RequestScheduler.getNumberOfAvailableRequests()).toEqual(RequestScheduler.maximumRequests);
-            });
```

Comment:
I have this here check that the ready promise is actually rejected. Otherwise that area won't have code coverage.